### ISSUE TITLE
The Swagger Page is throwing a JavaScript Error

### ISSUE THREAD

**Issue by rodrigojv** [FundacionParaguaya/FP-PSP-SERVER#149]
When accessing the swagger page: https://testing.backend.povertystoplight.org/swagger/index.html

It's not possible to navigate through the endpoints due to an JS error in the page.

Here is a snippet of the error pull out from the Developer Console:
```javascript
n
$ref
:
"#/definitions/UserDetailsDTO"
baseDoc
:
undefined
fullPath
:
(7) ["paths", "/api/v1/applications", "get", "parameters", "4", "schema", "$ref"]
level
:
"error"
line
:
1
message
:
"Could not resolve reference: #/definitions/UserDetailsDTO"
originalError
:
undefined
path
:
"paths./api/v1/applications.get.parameters.4.schema.$ref"
pointer
:
"/definitions/UserDetailsDTO"
source
:
"resolver"
type
:
"thrown"
stack
:
"JSONRefError: Could not resolve reference: #/definitions/UserDetailsDTO↵    at Object.plugin
```

**Comment by rodrigojv** [comment]
Maybe we should upgrade the swagger version in the project.

**Comment by pA1nD** [comment]
## The problem:
Swagger UI is showing the endpoints but not displaying the details for each endpoint.

## The solution:
We should try to update the Swagger UI or look for another way to solve the problem.

**Comment by rodrigojv** [comment]
Fixed by #235
